vispy/plot/plotwidget.py

```diff
@@ -42,9 +42,9 @@ def __init__(self, *args, **kwargs):
         super(PlotWidget, self).__init__(*args, **kwargs)
         self.grid = self.add_grid(spacing=0, margin=10)
         self.title = scene.Label("", font_size=16)
-        self.title.stretch = (1, 0.1)
-        self.grid.add_widget(self.title, row=0, col=2)
-        self.view = self.grid.add_view(row=1, col=2, border_color='grey')
```

Comment:
you're going to put all these back eventually, right? I think the existing API of setting `stretch` is good, maybe we should replace `min_width` by `min_size` to be consistent, though.